File: Src/CSharpier.Rider/src/main/java/com/intellij/csharpier/Semver.java
package com.intellij.csharpier;

import java.lang.module.ModuleDescriptor;

public class Semver {

    public static boolean gte(String version, String otherVersion) {
        return (
            ModuleDescriptor.Version.parse(version).compareTo(
                ModuleDescriptor.Version.parse(otherVersion)
            ) >=
            0
        );
    }
}

File: Src/CSharpier.VisualStudio/CSharpier.VisualStudioShared/Semver.cs
namespace CSharpier.VisualStudio
{
    using System;
    using NuGet.Versioning;

    public static class Semver
    {
        public static bool GTE(string version, string otherVersion)
        {
            return SemanticVersion.Parse(version).CompareTo(SemanticVersion.Parse(otherVersion))
                >= 0;
        }
    }
}
